docker-compose.yml

```diff
@@ -93,6 +94,21 @@ services:
       XDEBUG_CONFIG: client_host=host.docker.internal discover_client_host=0 # Required for MacOS only.
       WP_PHPUNIT__TESTS_CONFIG: local/public/wp-tests-config.php
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      - discovery.type=single-node
+      - cluster.name=elasticsearch
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
```

Comment:
**question:** Should we add a comment explaining [what these do](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_disable_swapping)? Are they even necessary for local development where we don't care about performance?